ocaml-lsp-server/src/code_actions/action_annotate.ml

```diff
@@ -0,0 +1,63 @@
+open Import
+
+let action_kind = "annotate"
+
+let check_context doc pos_start =
```

Comment:
I'd choose a name that's closer to the property being checked. `check_typeable_context` or something.